src/betterbib/cli/doi2bibtex.py

```diff
@@ -47,4 +84,12 @@ def _get_parser():
         default=sys.stdout,
         help="output file (default: stdout)",
     )
+    parser.add_argument(
```

Comment:
Let's not create an extra cmd option for this. Just make it the default behavior.